angular-workspace/projects/example-client-app/src/app/app.module.ts

```diff
@@ -10,6 +10,7 @@ import { NimbleTextAreaModule, NimbleTextFieldModule, NimbleNumberFieldModule, N
     NimbleCardButtonModule, NimbleDialogModule, NimbleRadioGroupModule, NimbleRadioModule, NimbleSpinnerModule,
```

Comment:
@kjohn1922 looks ready to move out of draft, dunno if you were waiting on anything else

Reply:
I was waiting for sign-off from @mollykreis to move it out of draft, but I can go ahead and do that with your approval